### ISSUE TITLE
[Bug][A11Y][Filter Chip] Filter chip is not reached by keyboard focus or Switch Access focus

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#474]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

Samsung S22

### OS Version

Android 16

### App version

0.60 (491)

### Describe the issue

Filter chip is not reached by keyboard focus or Switch Access focus.
It works well on suggestion chip.

### Expected Behavior

Filter chip shall be reached by assistive tools as the suggestion chip

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (694)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 474
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (694)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 474
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
fixed for keyboard but still fails with Switch Access, focus doesn't go through the filter chip component

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (700)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 688
Related issue: #474
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (700)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 688
Related issue: #474
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
fixed
